setup.cfg

```diff
@@ -7,7 +7,7 @@ long_description_content_type = text/markdown
 packages = 
 	atools
 install_requires =
-	dataclasses>="0.6";python_version=="3.6"
+	dataclasses>="0.6";python_version>="3.6"
```

Comment:
If I recall correctly, this only installs the dataclasses pypi package if we're using Python 3.6. Python 3.7 and beyond have it in the standard library. Did this actually fail for you?

Reply:
Ah, no - I may have misinterpreted some warnings during installation.